api/src/main/java/org/openmrs/module/aijar/ugandaemr/PublicHoliday.java

```diff
@@ -0,0 +1,67 @@
+package org.openmrs.module.aijar.ugandaemr;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.openmrs.BaseOpenmrsData;
+import org.springframework.stereotype.Component;
+
+@Component
+@Entity(name = "public_holiday")
+@Table(name = "public_holiday")
+public class PublicHoliday extends BaseOpenmrsData{
```

Comment:
@reagan-meant You have placed this class in the package org.openmrs.module.aijar.ugandaemr instead of org.openmrs.module.ugandaemr as I earlier requested move it

Reply:
Done